weather@mockturtl/files/weather@mockturtl/3.0/settings-schema.json

```diff
@@ -8,6 +8,22 @@
   , "description": "Set the keybinding to call the menu"
   , "default": "<Super><Shift>w"
   },
+  "descriptions":
```

Comment:
I need an opinion on this section of the Settings, is it ok to do something like this?
It looks like this:
![image](https://user-images.githubusercontent.com/22377521/73191073-d49b6700-411e-11ea-9511-fcc362bb0878.png)

Reply:
Looks fine to me